test/integration.test.js

```diff
@@ -110,13 +113,7 @@ it(`should execute "ncc build web-vitals" with target config`, async () => {
   await new Promise(resolve => setTimeout(resolve, 5000));
 });
 
-afterAll(() => {
-  if (coverage)
-    process.exit(0);
-});
```

Comment:
Why is this necessary? I think it's causing issues in CI, I removed it for now.

Reply:
I don't recall why this was here. It was probably to fix a bug where the process was hanging. We'll see if it hangs once removed.

Reply:
Looks like all passed except the test coverage 

https://github.com/vercel/ncc/actions/runs/10862816229/job/30202179532?pr=1213

Let's add this back

Reply:
Alright, I added it back. Hopefully, all CI issues are resolved, but I feel like it will still not work for "Node 20 and ubuntu-latest", with test coverage.